website/templates/emails/pending_retraction_admin.txt.mako

```diff
@@ -0,0 +1,15 @@
+Hello ${user.fullname},
+
+
+I just wanted to let you know, ${initiated_by} has initiated a retraction for the following registration: ${registration_link}
```

Comment:
Maybe I am misunderstanding what this is - don't we have a retraction alert for admins already that explains what will occur and offers and opportunity to approve or disapprove?